upsertGitHubTag-f79c4f36-3e61-43f4-8f6c-0b2e0e9774d7/deployment/index.js

```diff
@@ -117,7 +148,27 @@ function processEvent(event, callback) {
             }
         });
     } else {
-        console.log('Ignoring deletion push event')
+        console.log('Valid push event (delete)');
+        const repository = body.repository.full_name;
+        const gitReference = body.ref;
+
+        path += "workflows/github";
+        
+        deleteEndpoint(path, repository, gitReference, (response) => {
+            console.log(response);
```

Comment:
This callback handler is exactly the same for the POST and the DELETE (I think). If it is, you could make it a common function that both invoke.